apps/s_cb.c

```diff
@@ -26,10 +26,8 @@
 
 #define COOKIE_SECRET_LENGTH    16
 
-int verify_depth = 0;
-int verify_quiet = 0;
-int verify_error = X509_V_OK;
-int verify_return_error = 0;
+VERIFY_CB_ARGS verify_args = { 0 };
```

Comment:
I know, it will throw a warning " missing fields initializer" , when enabled ...
There is already enough of it in current base-code ...